sidebars.js

```diff
@@ -62,6 +62,26 @@ const sidebars = {
     },
     'metrics',
     'examples',
+    {
+      type: 'category',
+      label: 'Tutorials',
+      collapsible: true,
+      collapsed: false,
+        items: [
+          {
+            type: 'category',
+            label: 'Minimal Working Examples',
```

Comment:
An additional section has been added because we plan to add tutorials that are not minimal working examples. Like Jellyfish Dashboard introduction or in depth tutorial of particular react hook